demisto_sdk/commands/common/constants.py

```diff
@@ -23,6 +23,7 @@
 TOOLS_DIR = 'Tools'
 RELEASE_NOTES_DIR = 'ReleaseNotes'
 TESTS_DIR = 'Tests'
+DOC_FILES_DIR = 'doc_files'
```

Comment:
Shouldn't this be uppercased?
And why not DocFiles?

Reply:
@reutshal this is the convention in the repo